### ISSUE TITLE
Viewport rules repeated in generated CSS

### ISSUE THREAD

**Issue by fofr** [alphagov/govuk_frontend_toolkit#229]
The rules found at https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_conditionals.scss#L28-L34 are being repeated every time `_conditionals` is used.

eg Search for `-ms-viewport` in https://assets.digital.cabinet-office.gov.uk/static/govuk-template-b5d680ad6607cf5441a22a3a533e92b40b15b8f6b9708dbdfc5fc6da71949113.css

```css
@-ms-viewport {
  width: device-width;
}

@-o-viewport {
  width: device-width;
}
```

**Comment by jonhurrell** [comment]
We've mitigated this issue on a project build by piping our sass stream through [node-sass-import-once](https://github.com/at-import/node-sass-import-once).

**Comment by fofr** [comment]
Fixed by #272
